web/models/queries.ex

```diff
@@ -23,6 +24,16 @@ defmodule Entice.Web.Queries do
     end
   end
 
+  def get_account_by_name(name) do
```

Comment:
Yes, I call the 'def get_account_id(name) do' function in it so I reused the same parameter.